docs/OV_Runtime_UG/supported_plugins/BATCH.md

```diff
@@ -0,0 +1,36 @@
+# Automatic Batching Plugin {#openvino_docs_IE_DG_supported_plugins_AUTO_BATCH}
+
+## (Automatic) Batching Plugin Execution (C++)
+
+@sphinxdirective
+.. raw:: html
+
+    <div id="switcher-cpp" class="switcher-anchor">C++</div>
+@endsphinxdirective
+
+The BATCH device is a preview of the new, special "virtual" device in the OpenVINO™ toolkit. It performs on-the-fly automatic batching (i.e. grouping inference requests together) to improve device utilization, with no programming effort from the user side.
+Gathering of the inputs (as well as scattering the outputs) from the individual inference requests that is needed to constitute the batch happens in the completely transparent way, so the application code is left intact. The major pre-requisite for the application to enjoy an associated performance improvements is *running many inference requests simultaneously*.
+So the feature primarily targets existing codes that are written for inferencing many requests (each instances with batch size of 1).
+  
+Since the batching is straightforward way of leveraging the GPU compute power (and save on communication overheads) the automatic batching is _implicitly_ triggered on the GPU when the 'THROUGHPUT' hint is used.
+Alternatively (in the legacy apps that are not akin to the notion of the hints) you may want to use the explicit device notion like 'BATCH:GPU'. In both cases the actual batch size selection happens on the per-model basis (also respecting things like on-device memory for the dGPUs).
+
+Notice that _automatic batch size selection_ assumes that the application queries the OPTIMAL_NUMBER_OF_INFER_REQUESTS to create and runs simultaneously the returned number of requests. This is very important, as there is a "timeout" value so that (when no enough inputs have been collected) the execution transparently falls back to execution the requests individually. Configuration-wise this is AUTO_BATCH_TIMEOUT property, please see below.
+The timeout, which adds itself to the execution time of the requests, heavily penalizes the performance. To avoid it, in the cases when your parallel slack is bounded (e.g. the application processes only 4 video-streams, so batch larger than 4 doesn't make sense) make sure to give the OpenVINO an additional hint the OV on this.
+When the performance hint is hint equip it with the additional PERFORMANCE_HINT_NUM_REQUESTS configuration key set to "4". This is the most future-proof way to communicate the limitations on the parallelism. E.g. for the GPU that will limit the batch size, for the CPU, the number of inference streams <link> accordingly, so each device uses the PERFORMANCE_HINT_NUM_REQUESTS when converting the hint to the actual device configuration options.
```

Comment:
use new properties instead of `PERFORMANCE_HINT_NUM_REQUESTS ` and others

Reply:
yes, and even better add links to the property's doxygen (whihc as we agreed should have the old name in the comments)- avoiding actual property name ( whihc may need update if we rename smth)